examples/huggingface/pytorch/language-modeling/pruning/run_clm_sparsegpt.py

```diff
@@ -45,13 +45,15 @@ def skip(*args, **kwargs):
     default_data_collator,
     get_scheduler,
 )
+
 from transformers.utils import check_min_version, get_full_repo_name, send_example_telemetry
 from transformers.utils.versions import require_version
 from timers import CPUTimer, GPUTimer
 from intel_extension_for_transformers.transformers.pruner import (WeightPruningConfig,
                                                                   prepare_pruning,
                                                                   model_slim,
                                                                   parse_auto_slim_config)
+from intel_extension_for_transformers.llm.evaluation.lm_eval import evaluate as lm_evaluate
```

Comment:
we don't need to import gptj ```from intel_extension_for_transformers.transformers```?

Reply:
import AutoModelForCausalLM is enough, here is the mapping: https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/transformers/modeling/modeling_map.py